### ISSUE TITLE
Clarification of MLE/OLS/MSE

### ISSUE THREAD

**Issue by nandanrao** [Icedgarr/Open_Guide_Statistical_Tools#1]
In [Regularisation Opening](https://github.com/Icedgarr/Open_Guide_Statistical_Tools/blob/master/Statistics-open-sheet.Rmd#why-are-we-doing-all-of-this-anyway):

MLE doesn't minimize RSS unless the likelihood function is guassian, in which case it is the OLS. The squared bias is squared loss, which again, is the MLE loss function only for Gaussian distributions, or OLS. 

I'm just suggesting that It's worth clarifying that MSE as a framework for thinking about regularization comes from normality, because when your predictive distribution is normal the KL divergence from model -> data (decreased by maximizing entropy and maximizing likelihood) can be described in shorthand (maximize entropy = minimize variance (down to 1), maximize likelihood = minimize squared bias/error), and the purpose of regularization is always to curtail the tendencies of MLE, which is the divergence from data -> model, by considering things in both directions (model -> data & data -> model). This is what prevents the MLE, which in the normal case will always try and plop the mode in the middle and just crank up the variance to deal with everything else, from doing exactly that. 

In other words, by weighing BOTH how the model fits the data as well as how the data fits the model, in the normal case this prevents us from dropping a gaussian with huge variance into the middle of the data, but the ideas are equally applicable outside the realm of gaussianity.
